src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,11 +5,27 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private double bladeWidth;
+    private int enginePower;
```

Comment:
You should never want to expose the object fields directly. They should be accessed through special methods (getters and setters). Please add private access modifiers to your `bladeWidth` and `enginePower` fields.